### ISSUE TITLE
Issue with k8s.io/docs/concepts/configuration/assign-pod-node/

### ISSUE THREAD

**Issue by joelachance** [kubernetes/website#14264]
**This is a Bug Report**

**Problem:**
Potential Typo. Versions > 1.1 are referenced.

**Proposed Solution:**
Update versions in docs.

**Page to Update:**
https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels

Kubernetes Version: v1.14

Additional Information: It looks like this occurs 3 times. v1.4 is called out twice, and v1.2 is called out once. It seems like those versions aren't stable yet based on the published documentation, so I'm assuming it might be a typo.

Forgive this if I'm wrong :) Have a good day Kubernetes team!

**Comment by sftim** [comment]
[Node affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature) ought to use a Hugo feature state-shortcode so that the page follows the website style guide.

It's stable since Kubernetes v1.6 so the shortcode would look like:
```markdown
{{</* feature-state for_k8s_version="v1.6" state="stable" */>}}
```

**Comment by sftim** [comment]
Looks like this is fixed, via #14271.
/close
